manifests/server.pp

```diff
@@ -1,4 +1,5 @@
 class amanda::server (
+  String            $ensure                   = 'present',
```

Comment:
Same here: maybe we can use a type `Enum['present', 'absent']`?

Reply:
Okay, I changed the type to an Enum.